scss/civicrm/dashboard/_dashboard.scss

```diff
@@ -0,0 +1,57 @@
+$dashboard-dashlets-margin: 10px;
+
+.crm-inactive-dashlet-fieldset {
+  legend {
+    background: none;
+  }
+
+  .crm-hover-button {
+    color: $brand-primary;
+
+    &:hover {
+      color: darken($brand-primary, $crm-darken-percentage);
+    }
+  }
+}
+
+#civicrm-dashboard {
+  .crm-dashboard-droppable {
+    margin-left: $dashboard-dashlets-margin / 2;
+    margin-right: $dashboard-dashlets-margin / 2;
+  }
+
+  .crm-dashboard-droppable:last-child {
+    margin-right: 0;
+  }
+
+  .crm-dashlet {
+    margin: $dashboard-dashlets-margin 0 0;
+  }
+
+  .crm-inactive-dashlet,
+  .crm-dashlet {
+    border-radius: $border-radius-small;
+  }
+
+  .crm-dashlet-content {
+    border-radius: 0 0 $border-radius-small $border-radius-small;
+    padding: 0;
+  }
+
+  .crm-dashlet-header {
+    background: $gray-dark;
+    border-radius: $border-radius-small $border-radius-small 0 0;
+    padding: 17px 10px;
+  }
+
+  .crm-dashlet-header h3 {
+    border: 0;
+    font-size: $font-size-h3;
+    line-height: 135%;
```

Comment:
Why this 135% value?

Reply:
This value comes from the core style:
https://github.com/civicrm/civicrm-core/blob/master/css/civicrm.css#L235

Will move it to a variable under CRM.